src/agproxysecret/secret.go

```diff
@@ -129,7 +129,7 @@ func (agProxySecretGenerator *ActiveGateProxySecretGenerator) proxyUrlFromUserSe
 func parseProxyUrl(proxy string) (host string, port string, username string, password string, err error) {
 	proxyUrl, err := url.Parse(proxy)
 	if err != nil {
-		return "", "", "", "", err
+		return "", "", "", "", fmt.Errorf("could not parse proxy URL")
```

Comment:
use `errors.New()` from the `pkg/errors` library

Reply:
Done